src/services/CLIService.ts

```diff
@@ -240,6 +245,16 @@ export class CLIService implements IService{
             default: false
           });
     }
+
+    private enableDebugOption(yargs: Argv<{}>): void {
+        yargs.option('enable-debug', {
+            alias: 'g',
```

Comment:
Why `g`? IMO we should remove it, if it's not relevant to the options itself. Down the road we might need g for something relevant to the letter itself, but it will be taken by this. Let the user specify enable-debug, if they want to use it.